Example/Styles/Sources/AppDelegate.swift

```diff
@@ -46,7 +46,7 @@ let body = TextStyle(
 )
 
 let rounded = LayerStyle(
-        .cornerRadius(10),
+        .roundCorners([.topLeft, .bottomRight], radius: 10),
```

Comment:
How about to keep both options ? most cases are fully rounded, huh ?

Reply:
for all rounded corners you create following 
```
.roundCorners(.allCorners, radius: 10)
```